packages/git/src/browser/git-scm-provider.ts

```diff
@@ -415,14 +415,14 @@ export class GitScmProvider implements ScmProvider {
             fileText = `${paths.length} files`;
         }
         return new ConfirmDialog({
-            title: 'Discard changes',
+            title: nls.localize('vscode.git/package/command.clean', 'Discard changes'),
```

Comment:
minor: I hadn't noticed before but I believe we should update to the following:

```suggestion
            title: nls.localize('vscode.git/package/command.clean', 'Discard Changes'),
```